packages/web/utils/__tests__/version-history.test.ts

```
@@ -0,0 +1,342 @@
+
+// Mock Supabase client
+const mockSupabase = {
+  rpc: jest.fn(),
+  from: jest.fn(() => ({
+    select: jest.fn(() => ({
+      eq: jest.fn(() => ({
+        eq: jest.fn(() => ({
+          single: jest.fn(),
+        })),
+        order: jest.fn(() => ({
+          then: jest.fn(),
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)